signature/types.go

```diff
@@ -112,6 +113,11 @@ type SignRequest struct {
 	// TSARootCAs is the set of caller trusted TSA root certificates
 	TSARootCAs *x509.CertPool
 
+	// RevocationTimestampingValidator is used for timestamping certificate
+	// chain revocation check after signing.
+	// When present, only used when timestamping is performed.
+	RevocationTimestampingValidator revocation.Validator
```

Comment:
Will `TSARevocationValidator` be a better name and consistent?

Reply:
Updated.